transactor/src/test/scala/io/mediachain/transactor/JournalBlockchainSpec.scala

```diff
@@ -35,26 +37,30 @@ object JournalBlockchainSpec extends io.mediachain.BaseSpec
   
   def hasEmptyBlock = {
     val context = JournalBlockchainSpecContext.context
-    val block = context.dummy.client.submit(JournalCurrentBlock()).join()
+    val bop = context.dummy.client.currentBlock
+    val block = Await.result(bop, timeout)
     (block.index must_== 0) and
     (block.chain must beNone) and
     (block.entries must beEmpty)
   }
   
   def accumulateBlock = {
     val context = JournalBlockchainSpecContext.context
-    val res = context.dummy.client.submit(JournalInsert(Entity(Map()))).join()
+    val op = context.dummy.client.insert(Entity(Map()))
+    val res = Await.result(op, timeout)
     res.foreach((entry: CanonicalEntry) => {
       context.entries += entry
       context.entityRef = entry.ref
     })
     val ref = context.entityRef
     for (x <- 1 to 5) {
```

Comment:
i kind of prefer the for loops when it is a multiline block, as I don't have to write the fat arrow.